src/bundle/Resources/views/parts/page_title.html.twig

```diff
@@ -1,13 +1,13 @@
 {% if title is defined %}
     <div class="ez-page-title pt-3">
-        <h2>
+        <h1>
```

Comment:
Instead of changing the header types we should use CSS classes to style the elements, so we won't have to care about the type of HTML element in our selectors. Our app is not built in a semantic way because there's no point doing it here. Google robots will never crawl the app ;)

Reply:
I disagree. Even if Google does not crawl our app, we should follow the basic recommendations of web accessibility.
This code is public and should be an example for our community and other developers. 
I would rather have `h1` than `h2` for the page title.

Reply:
Our app is not accessible then.